packages/core/src/components/Tooltip/Tooltip.module.scss

```diff
@@ -15,12 +13,35 @@ $tooltip-z-index: 9999999999999999;
   &.withMaxWidth {
     --tooltip-max-width: 240px;
   }
-}
 
-.tooltip.paddingSizeMd {
-  border-radius: var(--border-radius-medium);
-  padding: var(--spacing-medium);
-  @include font-input();
+  .image {
+    display: block;
+    position: relative;
+    border-top-right-radius: 2px;
+    border-top-left-radius: 2px;
+    padding: 2px 2px 0 2px;
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+    min-width: var(--tooltip-max-width);
+    z-index: 1;
+  }
+
+  .title {
+    @include vibe-text(text2, bold);
+  }
+
+  .content {
+    word-break: break-word;
+    display: inline-block;
+    padding: var(--tooltip-padding, #{var(--spacing-small) var(--spacing-medium)});
```

Comment:
what? please explain the magic, wasn't familiar with this syntax, lol

Reply:
I actually copied it from line 9 here, but it's basically a variable with a default value, and the hash is to interpolate a SCSS variable, though we use only a CSS variable so it's redundant, let's remove it safely in next major